lib/asciidoctor/csa/converter.rb

```diff
@@ -77,15 +80,15 @@ def title_validate(root)
       end
 
       def makexml(node)
-        result = ["<?xml version='1.0' encoding='UTF-8'?>\n<csand-standard>"]
+        result = ["<?xml version='1.0' encoding='UTF-8'?>\n<csa-standard>"]
         @draft = node.attributes.has_key?("draft")
         result << noko { |ixml| front node, ixml }
         result << noko { |ixml| middle node, ixml }
-        result << "</csand-standard>"
+        result << "</csa-standard>"
```

Comment:
@ronaldtse @opoudjis let me know if this change should not be done, BTW I updated XML grammar to allow both names `csand-standard` and `csa-standard` (for backward compatibility)

Reply:
@CAMOBAP795 let's change it.